pyproject.toml

```diff
@@ -1,12 +1,84 @@
-[tool.black]
+[tool.ruff]
+exclude = [
+    ".git",
+    ".ruff_cache",
+    ".venv",
```

Comment:
Using `extend-exclude` makes these obsolete (and is recommenced by upstream)

https://docs.astral.sh/ruff/settings/#exclude